build.js

```diff
@@ -344,7 +344,15 @@ const flattenIDL = (specIDLs, customIDLs) => {
     (dfn) => dfn.type !== 'includes' && dfn.type !== 'interface mixin'
   );
 
-  return {ast, globals};
+  // Get all possible scopes
```

Comment:
This part isn't quite right. We don't need to strip 'GlobalScope' if we use the identifiers in the Global extended attribute. Here's what I got after some tinkering:
```js
  const scopes = new Set();
  for (const dfn of ast) {
    const attr = getExtAttr(dfn, 'Global');
    if (!attr) {
      continue;
    }
    // Special case RTCIdentityProviderGlobalScope since it doesn't use the
    // Global extended attribute correctly:
    // https://github.com/w3c/webrtc-identity/pull/36
    if (dfn.name === 'RTCIdentityProviderGlobalScope') {
      scopes.add('RTCIdentityProvider');
      continue;
    }
    switch (attr.rhs.type) {
      case 'identifier':
        scopes.add(attr.rhs.value);
        break;
      case 'identifier-list':
        for (const {value} of attr.rhs.value) {
          scopes.add(value);
        }
        break;
      default:
        throw new Error(`Unable to parse Global extended attribute of ${dfn.name}`);
    }
  }
```

Can you add that and adapt the tests?